lib/app/modules/addOrUpdateAlarm/views/label_tile.dart

```diff
@@ -57,6 +58,27 @@ class LabelTile extends StatelessWidget {
             onChanged: (text) {
               if (text.trim().isEmpty) {
                 controller.labelController.text = "";
+                if (text.isNotEmpty) {
+                  Flushbar(
+                    messageText: Text(
+                      "Please enter valid characters",
```

Comment:
We should change the message to something that isn't so ambiguous, because this would hint that space is invalid as a character :P

Reply:
Yes, you're right. I'll be changing it to something meaningful.